Dockerfile.prod

```diff
@@ -65,10 +65,11 @@ RUN touch /var/log/stats.log /var/log/dump_create.log && chown listenbrainz:list
 RUN touch /etc/service/cron/down
 
 # Node
-RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
-RUN apt-get install -y nodejs
-RUN npm install -g less@2.7.2
-RUN npm install less-plugin-clean-css
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get install -y nodejs && rm -rf /var/lib/apt/lists/*
+COPY ./package.json /code/listenbrainz/
```

Comment:
If I change my package.json, is that change not going to be picked up by the `COPY`command at line 47 ?
Does line 47 need to be modified to not copy the JS-related files?
https://github.com/metabrainz/listenbrainz-server/pull/504/files#diff-0e9722a692af2b244674b56146a1ecd3R47

Reply:
you're right, normally specific copies should be done earlier in the file (package.json, requirements.txt) and bulk copies (COPY .) later in the file. This way, changes to other files in the repository won't cause all the future commands to be re-run.
Having said this, I guess this is the production dockerfile, so perhaps it's not as much as an issue because this isn't run as often, but I agree that this file should be re-organised a bit.